Dockerfile.template

```diff
@@ -9,15 +9,18 @@ ENV ME_CONFIG_MONGODB_URL="mongodb://mongo:27017" \
 
 ENV MONGO_EXPRESS_VERSION={{ .version }}
 
+WORKDIR /home/node
+
+ENV YARN_VERSION="4.0.0-rc.26"
```

Comment:
Will we only need this temporarily to get 1.0.1 out? Or do we need to pin the yarn version long term?

Cosmetic: we should keep env vars together in the compose file where possible.

Reply:
I thought I fixed it but it didn't work. I don't know if yarn 4 is required but it seems that version has a patch that doesn't apply on install

Reply:
@shakaran @rtritto would either of you have time to have a look?

Reply:
We can import once the versions (_yarn_ and _mongo-express_) from _.json_ file ([souce](https://stackoverflow.com/questions/49681984/how-to-get-version-value-of-package-json-inside-of-dockerfile)).